src/core/agent/services/identifierService.ts

```diff
@@ -127,21 +126,19 @@ class IdentifierService extends AgentService {
 
   @OnlineOnly
   async createIdentifier(
-    metadata: Omit<
-      IdentifierMetadataRecordProps,
-      "id" | "createdAt" | "signifyName"
-    >
+    metadata: Omit<IdentifierMetadataRecordProps, "id" | "createdAt">
   ): Promise<CreateIdentifierResult> {
     const startTime = Date.now();
 
     if (!identifierTypeThemes.includes(metadata.theme)) {
       throw new Error(`${IdentifierService.THEME_WAS_NOT_VALID}`);
     }
 
-    const signifyName = uuidv4();
-    const operation = await this.props.signifyClient
-      .identifiers()
-      .create(signifyName); //, this.getCreateAidOptions());
+    let name = `${metadata.theme}:${metadata.displayName}`;
+    if (metadata.groupMetadata) {
+      name = `${metadata.theme}:${metadata.groupMetadata.groupId}:${metadata.displayName}`;
+    }
+    const operation = await this.props.signifyClient.identifiers().create(name); //, this.getCreateAidOptions());
```

Comment:
is prettier running on commit in your PRs? (Jorge posted in the channel about this recently)

Reply:
prettier was running before PR was created